swan-lake/integration-tools/health-tool.md

```diff
@@ -420,5 +421,224 @@ Follow the steps below to use the generated API templates by running the cloned
+   }
+   ```
+
+## CDS template generation
+
+A Ballerina service template can be generated from the given CDS hook definitions. This template will also include basic functionalities such as validation, prefetch, etc., and it will facilitate the developers' implementation of the required connection with the external decision support system to run the CDS server. The generated Ballerina service project will be written into the provided output location.
+
+Supported CDS version: [2.0](https://cds-hooks.hl7.org/2.0)
+
+### Template generation usage
+
+```
+$ bal health cds
+            [--org-name] <template-organization-name>
+            [--package-name] <name-of-the-package>
+            [--package-version] <version-of-the-package>
+            [-o | --output] <output-location>
+            [-i | --input] <cds-hook-definitions-file-path>
+```
+
+### CDS template generation command options
+
+| Command option      | Description                                                                                                                                                                                                                                                                                                                                                                                                                           | Mandatory/Optional |
+| ------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------ |
+| `-i, --input`       | Users need to define the CDS hook definitions in the input file. The given CDS hooks input will be used to generate the Ballerina service. Only the TOML file type is acceptable.  | Mandatory          |
+| `--org-name`        | Organization name of the Ballerina template to be generated. For more information, see <a href="https://ballerina.io/learn/package-references/#the-org-field" target="_blank"> the <code>org</code> field</a>.                                                                                                                                                                                                         | Optional           |
+| `--package-name`    | Name of the Ballerina package to be generated. The package name can be explicitly set using this argument. If not specified, the default name `health.fhir.templates.crd` will be used to construct the name of the package.                                                                                                                                                                                                         | Optional           |
```

Comment:
```suggestion
| `--package-name`    | Package name to be used for the generated Ballerina package. If not specified, `health.fhir.templates.crd` will be used to construct the name of the package.                                                                                                                                                                                                         | Optional           |
```

Used or will it be exactly that? If the latter, would say "If not specified, `health.fhir.templates.crd` will be used as the package name."